File: Łatwe/Java/Harry and big doughnuts/HarryAndBigDoughnuts.java
import java.util.Scanner;

class HarryAndBigDoughnuts {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int t = in.nextInt();
        for (int i = 0; i < t; i++) {
            int c = in.nextInt();
            int k = in.nextInt();
            int w = in.nextInt();
            if (c * w <= k)
                System.out.println("yes");
            else
                System.out.println("no");
        }
    }
}

File: Łatwe/C++/Harry and big doughnuts/HarryAndBigDoughnuts.cpp
#include <iostream>

int main() {
    int t;
    std::cin >> t;
    for (int i = 0; i < t; i++) {
        int c;
        int k;
        int w;
        std::cin >> c >> k >> w;
        if (c * w <= k)
            std::cout << "yes\n";
        else
            std::cout << "no\n";
    }
}
